runtime/gc_base/GCExtensions.cpp

```diff
@@ -267,3 +267,13 @@ MM_GCExtensions::computeDefaultMaxHeap(MM_EnvironmentBase *env)
 
 	memoryMax = MM_Math::roundToFloor(heapAlignment, memoryMax);
 }
+
+MM_OwnableSynchronizerObjectList *
+MM_GCExtensions::getOwnableSynchronizerObjectListsExternal(J9VMThread *vmThread)
+{
+	if (isConcurrentScavengerInProgress()) {
+		vmThread->javaVM->memoryManagerFunctions->j9gc_modron_local_collect(vmThread, J9MMCONSTANT_IMPLICIT_GC_COMPLETE_CONCURRENT);
```

Comment:
This is within GC code. No need to go through inter-module API. Just inline here whatever j9gc_modron_local_collect does. Then, you don't need to change the signature of j9gc_modron_local_collect and other callers of j9gc_modron_local_collect will not be affected.